src/nvim/option.c

```diff
@@ -2440,11 +2440,9 @@ did_set_string_option (
     if (briopt_check(curwin) == FAIL)
       errmsg = e_invarg;
   }
```

Comment:
Hhmm. Now, putting the comment inside the block, the outer block is now inconsistent with its weird styling.